internal/pkg/pm2/pm2.go

```diff
@@ -76,8 +83,43 @@ func (m *processManager) pm2(p Process, cmdName string) error {
 		"pm2",
 		cmdArgs...,
 	)
+	m.notifyf("executing command '%v'", strings.Join(cmd.Args, " "))
+	if m.verbose {
+		chromeStdErr, err := cmd.StderrPipe()
```

Comment:
```suggestion
		processStdErr, err := cmd.StderrPipe()
```

It may also be `unoconv` 😄